app/assets/javascripts/admin/templates/components/admin-theme-editor.hbs

```diff
@@ -0,0 +1,94 @@
+<div class='edit-main-nav admin-controls'>
+    <nav>
+    <ul class='nav nav-pills target'>
+        {{#each visibleTargets as |target|}}
+        <li>
+            {{#link-to editRouteName
+            theme.id 
+            target.name 
+            fieldName 
+            replace=true 
+            title=field.title 
+            class=(if target.edited 'edited' 'blank')
+            }}
+            {{#if target.error}}{{d-icon 'exclamation-triangle'}}{{/if}}
+            {{#if target.icon}}
+                {{d-icon target.icon}}
+            {{/if}}
+            {{i18n (concat 'admin.customize.theme.' target.name)}}
+            {{/link-to}}
+        </li>
+        {{/each}}
+        <li>
+        <a {{action "toggleShowAdvanced"}} 
+            class='no-text' 
+            title="{{i18n (concat "admin.customize.theme." (if showAdvanced "hide_advanced" "show_advanced"))}}"
+        >
+            {{#if showAdvanced}}
+            {{d-icon "angle-double-left"}}
```

Comment:
I think we could use the ember if helper here.

`{{d-icon (if showAdvanced "angle-double-left" "angle-double-right")}}`